diff --git a/node_modules/vitest/dist/chunks/cli-api.BQ-bjcRi.js b/node_modules/vitest/dist/chunks/cli-api.BQ-bjcRi.js
index 55fb6b0..6a4b290 100644
--- a/node_modules/vitest/dist/chunks/cli-api.BQ-bjcRi.js
+++ b/node_modules/vitest/dist/chunks/cli-api.BQ-bjcRi.js
@@ -7171,7 +7171,7 @@ class VmThreadsPoolWorker extends ThreadsPoolWorker {
 	}
 }

-const WORKER_START_TIMEOUT = 5e3;
+const WORKER_START_TIMEOUT = 15e3;
 class Pool {
 	maxWorkers = 0;
 	workerIds = /* @__PURE__ */ new Map();
